app/views/results/_location.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="filter-form" data-qa="filters__location">
+  <h2 class="govuk-heading-s filter-form__title">
+    Location<span class="govuk-visually-hidden">:</span>
+  </h2>
+  <ul class="filter-form__value--list">
```

Comment:
@aliuk2012 what do you think? It's only there to make the styling work, there is only ever one item.

Reply:
We shouldn't include elements just for styling purposes. 

Having a `ul` suggests that there is more than one item selected. 🤔 Maybe we can leave `ul` in here for the time being and create another ticket to ensure if there is only one filter option selected for any of the fliters then we do not use `ul`. This will improve the semantic markup we are using on the page.